generate_keys.sh

```diff
@@ -3,15 +3,18 @@
 set -e  # Exit on error
 set -x  # Debug mode
 
+# Get prefix from command line argument, default to empty string if not provided
+PREFIX=$1
```

Comment:
you should validate that this is not empty

you should exit with an usage message if no parameters are provided

Reply:
Done